spec/models/review_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+  let(:review) { Review.new(name: "Arnold Schwarzenegger",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.